packages/form-js-editor/src/features/properties-panel/groups/ValidationGroup.js

```diff
@@ -235,6 +235,7 @@ function Min(props) {
     element: field,
     id,
     label: 'Minimum',
+    step: 'any',
```

Comment:
is this a standard config of the properties panel?

Reply:
It's a standard config of number fields as per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#step. The properties panel components are pretty thin wrappers around the basic HTML components.